Events/eventModels.js

```diff
@@ -0,0 +1,15 @@
+var mongoose = require('mongoose');
+
+var eventSchema = new mongoose.Schema({
+  category : String,
+  contact : String,
+  creator : String,
+  datetime : String,
+  description : String,
+  location : String,
+  title : String
+});
+
+var Event = mongoose.model('eventModel', eventSchema, 'events');
```

Comment:
If this is about ```mongoose.model(...)```, this is the same as reports.  I just followed some example code for reports, but I'm pretty sure it's this way because ```Schema``` is essentially a class with a constructor, but ```mongoose.model()``` is just a function that produces a model.

That being said, I do think that the model name (the first parameter of ```mongoose.model()```) should be called 'Event' for consistency with Reports.